libs/net/models/Areas/Services/Models/DataSource/DataLocationModel.cs

```diff
@@ -8,6 +10,15 @@ namespace TNO.API.Areas.Services.Models.DataSource;
 public class DataLocationModel : BaseTypeModel<int>
 {
     #region Properties
+    /// <summary>
+    /// get/set - The location type.
+    /// </summary>
+    public DataLocationType LocationType { get; set; }
+
+    /// <summary>
+    /// get/set - The connection settings.
+    /// </summary>
+    public Dictionary<string, object> Connection { get; set; } = new Dictionary<string, object>();
```

Comment:
Need these properties so that Ingest Services have access to it.  I don't love returning credentials this way in the future so we'll need to tighten up security for this endpoint.